.github/dependabot.yml

```diff
@@ -28,3 +28,18 @@ updates:
     open-pull-requests-limit: 99
     allow:
       - dependency-type: direct
+
+  - package-ecosystem: docker
+    directory: '/'
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 99
+    ignore:
+      - dependency-name: 'moritzheiber/ruby-jemalloc'
+        update-types:
+          # only suggest patch releases for ruby and needs to sync with .ruby-version
+          - 'version-update:semver-minor'
+      - dependency-name: 'node'
```

Comment:
Is there something preventing us modifying the Dockerfile to remove ARG usage so it works ? (like in https://github.com/mastodon/mastodon/pull/23552/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557)

Reply:
Not sure. I've also opened #23554, to just float at the major version, since the config files don't seem to pin down to the minor/patch release